Demo/ViewController.swift

```diff
@@ -72,7 +72,7 @@ extension ViewController: MessagesViewDelegate {
     func didTapRightButton() {
         let text = messagesView.inputText
         TestData.exampleMessageText.append(text)
-        messagesView.refresh()
+        messagesView.refresh(scrollDown: true)
```

Comment:
There is a setting for this feature: MessagesViewSettings::textInputScrollsToRecentMessage, and this one should be used and/or renamed if needed. My proposal is: `scrollsToLastMessage`.